index.html

```diff
@@ -1,238 +1,276 @@
+                    <li>CANOPY</li>
+                    <li><img src="Images/Counter.png" alt="Counter"></li>
+                    <li>Back End Dev</li>
+                    <li><img src="Images/Counter.png" alt="Counter"></li>
+                    <li>2015</li>
+                </ol>
+
+                <p>A daily selection of privately personalized reads;no accounts or sign-ups required.</p>
+
+                <ul>
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScipt</li>
+                </ul>
+                <button type="button" class="button"> See Project</button>
+            </div>
+        </div>
+
+
+        <div class="project">
+            <img src="Images/SnapshootPortfolio3.png" alt="SnapshootPortfolio3" class="img1">
+            <div>
+                <h2>Multi-Post <br class="br">Stories</h2>
+
+                <ol>
+                    <li>CANOPY</li>
+                    <li><img src="Images/Counter.png" alt="counter"></li>
+                    <li>Back End Dev</li>
+                    <li><img src="Images/Counter.png" alt="counter"></li>
+                    <li>2015</li>
+                </ol>
+
+                <p>A daily selection of privately personalized reads;no accounts or sign-ups required.</p>
+
+                <ul>
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScipt</li>
+                </ul>
+
+                <button type="button" class="button"> See Project</button>
+            </div>
+            <img src="./Images/SnapshootPortfolio-desktop4.png" alt="SnapshootPortfolio" class="img2">
+            <img src="Images/SnapshootPortfolio3.png" alt="SnapshootPortfolio3" class="img3">
+
+        </div>
+    </section>
 
-         
-             <li>
-                 <h2>Languages <i class="fas fa-angle-down"></i></h2>
-                 <ul class="tech-skills">
-                    <li>
-                       <div class="tech-skills-detail">
-                         <img src="Images/Ellipse1.png" alt="Javascript">
-                         <p>Javascript</p>
-                      </div>
-                    </li>
 
-                 <li>
-                   <div class="tech-skills-detail">
-                      <img src="Images/Ellipse2.png" alt="HTML">
-                      <p>HTML</p>
-                    </div>
-                 </li>
-            
+
+    <section class="about" id="about">
+        <div class="about1">
+            <h2>About <br> Myself</h2>
+
+            <p>Hello I'm a software developer! I can help <br class="br"> you build a product , feature or website <br class="br"> Look
+                through
+                some of my work and <br class="br"> experience!If you like what you see and <br class="br"> have a project you need coded,
+                don't
+                <br class="br"> hestiate to contact me.
+            </p>
+
+            <h3>LET'S CONNECT</h3>
```

Comment:
Fantastic job on making your portfolio interactive and responsive. 👏🏾 

There's just one slight issue with the responsiveness of the about section. On other viewpoints, the about section loses its margins and is not aligned with the languages section. Kindly look into that.

Your section:

![image](https://user-images.githubusercontent.com/38088818/167246217-9a1cde11-821c-4890-af51-ffb409dbbc63.png)

Figma:
![image](https://user-images.githubusercontent.com/38088818/167246276-f7f0b28f-23e3-49ef-9882-076ec4995449.png)